src/_includes/templates/nav/primary-nav.html

```diff
@@ -1,21 +1,28 @@
 {% import 'templates/nav/_vars-primary-nav.html' as vars with context %}
-<nav class="primary-nav sliding-nav_nav">
-    <ul class="primary-nav_top-level-list
-               wrapper
-               list__unstyled
-               list__horizontal
-               list-expanding">
-        {%- for nav_title, sub_nav_landing, sub_nav_title, sub_nav_media, sub_nav_items in vars.nav_items %}
-            <li class="list_item list-expanding_list-item">
+{% import 'macros/sub-nav.html' as sub_nav %}
+<nav class="primary-nav
+            js-primary-nav"
+     aria-expanded="false">
+    <ul class="primary-nav_menu-list
+               list__unstyled">
+        {%- for nav_title,
+                sub_nav_landing,
+                sub_nav_title,
+                sub_nav_media,
+                sub_nav_items in vars.nav_items %}
+            <li class="list_item
+                       primary-nav_item">
                 {%- if sub_nav_items|length > 0 %}
-                    {# TODO: Make this a <button>, it is not a real link. #}
-                    <a class="list_link list-expanding_trigger" href="#">
+                    {# TODO: Make this a button, it is not a real link. #}
+                    <a class="list_link
+                              primary-nav_link
+                              js-primary-nav_link"
+                       href="#">
```

Comment:
Remove href and add `u-link__disabled` class... unless the TODO means this shouldn't be handled like a disabled link.